src/components/sidebar/right-sidebar/datepicker/DateSlider.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+import './DateSlider.css';
+
+interface DSprops {
+    minPossibleValue: number; //minimum possible value for slider
+    maxPossibleValue: number; //maximum possible value for slider
+    values: Date[];
+    minDate: Date;
+    onChange: Function;
+}
+
+export default function DateSlider(props: DSprops){
+
+    const milliSecondsPerDay = 86400000;
+    const {minPossibleValue, maxPossibleValue, minDate} = props;
```

Comment:
Nit: It might be cleaner to destructure the props within the parameter on line 12 like `DateSlider({minPossibleValue, maxPossibleValue, minDate}: DSprops)`